server/services/licenceService.ts

```diff
@@ -70,43 +71,41 @@ export default class LicenceService {
   ) {}
 
   async createLicence(prisonerNumber: string, user: User): Promise<LicenceSummary> {
-    const [nomisRecord, deliusRecord] = await Promise.all([
-      this.prisonerService.getPrisonerDetail(prisonerNumber, user),
-      this.communityService.getProbationer({ nomsNumber: prisonerNumber }),
-    ])
+    const nomisRecord = _.head(await this.prisonerService.searchPrisonersByNomisIds([prisonerNumber], user))
```

Comment:
Changed the nomisRecord call to be a prisoner search API call, rather than prisoner API.
The prisoner API has a bunch of extra information that was never used, and using the prisoner search API meant I could use the same type as caseloadService, so the shared method was more tightly controlled.

Reply:
cool, and as we're not creating licences off the back of events there won't be any potential race conditions